src/main/java/com/cloudbees/jenkins/support/impl/AboutJenkins.java

```diff
@@ -707,29 +701,29 @@ private String getLabelString(Node n) {
             out.println();
             out.println("  * master (Jenkins)");
             out.println("      - Description:    _" +
-                    Markdown.escapeUnderscore(Util.fixNull(jenkins.getNodeDescription())) + "_");
+                    Markdown.escapeUnderscore(Util.fixNull(filter != null ? filter.filter(jenkins.getNodeDescription()) : jenkins.getNodeDescription())) + "_");
```

Comment:
We filter the pieces of information that may include some private information and avoid filtering the structure of the markdown